public/app/dsp-stages.js

```
@@ -0,0 +1,516 @@
+ *   Pass 1 (1–8)   : Pre-processing & gating
+ *   Pass 2 (9–16)  : Spectral ML masking
+ *   Pass 3 (17–24) : Voice EQ & formant shaping
+ *   Pass 4 (25–32) : Dynamics, limiting & output
+ *
+ * Usage:
+ *   import { applyStage } from './dsp-stages.js';
+ *   applyStage(stageName, mag, pha, params, sampleRate);
+ */
+
+// ─── Constants ────────────────────────────────────────────────────────────────
+const TWO_PI = 2 * Math.PI;
```

ESLint

[error] 'TWO_PI' is assigned a value but never used.

(no-unused-vars)